assimilation_code/modules/assimilation/assim_tools_mod.f90

```diff
@@ -541,8 +545,9 @@ subroutine filter_assim(ens_handle, obs_ens_handle, obs_seq, keys,           &
       ! Convert all my obs (extended state) variables to appropriate probit space
       call transform_to_probit(ens_size, obs_ens_handle%copies(1:ens_size, i), dist_for_obs, &
          obs_dist_params(i), probit_ens, .false., &
-         bounded_below, bounded_above, lower_bound, upper_bound)
-      obs_ens_handle%copies(1:ens_size, i) = probit_ens
+         bounded_below, bounded_above, lower_bound, upper_bound, ierr)
+      obs_probit_trans_ok(i) = (ierr == 0)
+      if(obs_probit_trans_ok(i)) obs_ens_handle%copies(1:ens_size, i) = probit_ens
```

Comment:
The added  if(obs_probit_trans_ok(i)) & at line 511 gives you the same answer as before, because of the bnrh 

fail -> `! Just return the original ensemble`
I guess this is for algorithm readability?